connaisseur/validators/notaryv1/trust_data.py

```diff
@@ -93,11 +94,28 @@ def validate_signature(self, keystore: KeyStore):
             sig = signature["sig"]
 
             try:
-                verify_signature(pub_key, sig, msg)
+                TrustData.__validate_signature_with_key(sig, msg, pub_key)
```

Comment:
Since we've in cosign gone away from public key, do you think it makes sense to go to key from pub_key here? I guess its mostly still PKC, but I think I've read stuff about x509 in #99 . Without having looked into it, pub_key may then be a certificate. While still mainly having the pubkey as interesting property, this would then be somewhat inaccurate

Since I don't see us implementing x509 soon though, we can also not do that now and stick with the name that better fits the current use case.

No strong stakes, just noticed. Thy choice